src/danog/MadelineProto/MTProtoTools/MinDatabase.php

```diff
@@ -88,6 +88,7 @@ public function init(): \Generator
             return;
         }
         \Amp\Loop::defer(function () {
+            yield new Delayed(1000);
```

Comment:
Really this is not needed, because a defer already makes the operation non-blocking

Reply:
On my 1 core VPS its makes difference. Probably mysql process other init requests faster. 

SELECT `key`, `value` FROM `{$this->table}` - this is heavy and takes few seconds before first result.